bears/general/CPDBear.py

```diff
@@ -73,12 +72,12 @@ def run(self, language: language, minimum_tokens: int=20,
             comparison.
         """
         for supported_lang in self.language_dict:
-            if supported_lang in language:
+            if supported_lang in programming_language:
                 cpd_language = self.language_dict[supported_lang]
                 break
         else:
-            self.err('This bear does not support files with the extension '
-                     "'{}'.".format(language))
+            self.warn('This bear has no support for ' +
```

Comment:
Why did you changed it to warning?

Reply:
Logging level error is generally used when something went wrong with the code. 
Warning indicates that code is working as expected, something went wrong due to wrong user input.
Hope this clears the doubt @newbazz